docs/elevenlabs-integration.md

```
@@ -0,0 +1,105 @@
+
+```bash
+export MISTRAL_API_KEY=sk-...
+
+# Start in public mode on a fixed port, allowing ElevenLabs origins
+./dir2mcp up --public --listen 0.0.0.0:8080 \
+  --allowed-origins "https://elevenlabs.io,https://api.elevenlabs.io"
+```
+
+The server prints:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)